app/assets/javascripts/orders.coffee

```diff
@@ -17,7 +17,7 @@ showOrderDialog = (orderId) ->
     error: (jqXHR, textStatus, errorThrown) ->
       alert "Error occurred"
 
-window.orderRowClicked = (event, row, element) ->
+orderRowClicked = (event, row, element) ->
```

Comment:
I think you should make this `window.` again and add back `data-click-fn` to the element... I have have styling that makes the pointer turn into a hand, and the behavior is then consistent with `data-href`.